### ISSUE TITLE
Internal NPM Output is being Leaked to the Console

### ISSUE THREAD

**Issue by X-Guardian** [Sceptre/sceptre-cdk-handler#8]
### Subject of the issue

Output from `npm` called by the internal prerequisite checker functions is leaking to the console.

### Your environment

- `sceptre-cdk-handler v1.0.0`

### Steps to reproduce

`sceptre launch cdk.yaml`

### Expected behaviour

```
[2023-01-05 12:43:50] - cdk - Launching Stack
[2023-01-05 12:43:51] - cdk - Stack is in the CREATE_COMPLETE state
[2023-01-05 12:43:51] - cdk - Updating Stack
[2023-01-05 12:44:14] - Publishing CDK assets
...
```

### Actual behaviour

```
[2023-01-05 12:43:50] - cdk - Launching Stack
[2023-01-05 12:43:51] - cdk - Stack is in the CREATE_COMPLETE state
[2023-01-05 12:43:51] - cdk - Updating Stack
sceptre-example@ C:\Users\Simon\Documents\GitHub\Sceptre\sceptre-cdk-handler\sceptre-example
└── (empty)

C:\Program Files\nodejs -> .\
└── cdk-assets@2.59.0

[2023-01-05 12:44:14] - Publishing CDK assets
...
```

### Resolution

Restore `capture_output=True` to the `subprocess.run` function calls for the prerequisite checker that were removed in PR #3

**Comment by jfalkenstein** [comment]
So an important caveat on the behavior here is that the NPM output is going to STDERR, not STDOUT. This is generally the pattern Sceptre follows to distinguish between command _logs_ and command _output_. This means that if you do something like `sceptre generate my-stack.yaml > template.yaml`, the output that goes to file will actually be only that which goes to stdout. The npm stdout is actually being redirected to stderr not to conflict with this behavior.

Personally, those logs are SUPER helpful when diagnosing odd behavior, trying to figure out what's failing, etc... It's also the same behavior as we use in the SAM handler. 

However... I think we could introduce a template handler argument for something like `quiet: True` or something like that that would bury the stdout logs rather than redirect to stderr. What would you think about that approach?

**Comment by jfalkenstein** [comment]
@X-Guardian any thoughts on this ^^?

**Comment by jfalkenstein** [comment]
I have _partially_ resolved this in #10 with this commit: https://github.com/Sceptre/sceptre-cdk-handler/pull/10/commits/4821965cf47c6d1ccaf02800f49ada103bae6453.

The specific "leakage" you included in your example is from when we check for whether those specific commands and assets exist. I've updated my branch to bury the output on those  when doing the checks. I think this will be ok. 

However, other log outputs from CDK/cdk-assets will still end up going to stderr as I mentioned above.

**Comment by X-Guardian** [comment]
Confirmed that the output from the `npm list` commands are now suppressed in #10, which is what I was talking about out.
